### PR TITLE
Move to EIP712 with WalletConnect 

### PR DESCRIPTION
**Pull request by bonustrack**

### CODE REVIEW

File: patches/@snapshot-labs+lock+0.1.100.patch [hunk 1]
```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts b/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts
```
Comment (samuveth) [inline patches/@snapshot-labs+lock+0.1.100.patch:1]: What do we need this patch for?

Comment (bonustrack) [reply patches/@snapshot-labs+lock+0.1.100.patch:1]: This is my attempt to fix the issue `require is not defined` but it's not working

Comment (samuveth) [reply patches/@snapshot-labs+lock+0.1.100.patch:1]: Ahh I see

**Comment by Abas0778** [comment]
> _ No description provided. _

**Pull request merged**